src/dcurl.c

```diff
@@ -126,6 +126,9 @@ int8_t *dcurl_entry(int8_t *trytes, int mwm, int threads)
 
 remote_pow:
     if (!doRemoteContext(&Remote_Context, pow_ctx)) {
+        freeRemoteContext(&Remote_Context, pow_ctx);
+        exitRemoteContext(&Remote_Context);
+        uv_sem_post(&notify_remote);
```

Comment:
Add comments about the termination of remote interface.

Reply:
Done.